ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -263,14 +263,23 @@ class DashboardPage extends Component {
       ],
     }
 
-    // this controls the auto group by behavior
     const interval = {
       id: 'interval',
-      type: 'constant',
+      type: 'autoGroupBy',
       tempVar: ':interval:',
-      resolution: 1000,
-      reportingInterval: 10000000000,
-      values: [],
+      label: 'automatically determine the best group by time',
+      values: [
+        {
+          value: '1000', // pixels
+          type: 'resolution',
+          selected: true,
```

Comment:
Selected is probably no longer the best word for this attribute. No need to update for this pr, but we should consider possible alternatives